LICENSE

```diff
@@ -1,4 +1,3 @@
-
                                  Apache License
```

Comment:
Can you please revert this change? `git checkout main LICENSE` should do the trick